api/src/database/database.py

```diff
@@ -120,7 +128,10 @@ def select(self, model: Type[Base] = None, query: Select = None,
                 query = query.limit(limit)
             if offset is not None:
                 query = query.offset(offset)
-            return self.session.execute(query).all()
+            results = self.session.execute(query).all()
+            if group_by:
+                return [list(group) for _, group in itertools.groupby(results, group_by)]
```

Comment:
If I am not mistaken, groupby needs a list sorted by the key (in our case stable_id).
I just want to make sure it's sorted. 
If it is, I suggest you put a comment like: "Results are sorted by stable_id since we used the ORDER BY in such and such place", or whatever mean was used to sort.

Reply:
@aronza to take this

Reply:
Thanks for pointing this out @jcpitre , I've added order by clause to queries using groupby